components/count-badge/README.md

```diff
@@ -25,6 +25,17 @@ The `d2l-count-badge` element is a web component to display a number count, eith
 | `max-digits`, default: `2` when `type="notification"` | Number | Optionally specify a digit limit, after which numbers are truncated. Defaults to two for `"notification"` type and no limit for `"count"` type.
 | `hide-zero`, default: `false` | Boolean | Optionally choose not to show the count badge when the number is zero. |
 
+### Accessibility Properties
+
+To make your `d2l-count-badge` accessible, use the following properties when applicable:
+
+| Attribute | Description |
+|--|--|
+| `description`, required | A description of the badge for screen-readers, to be used as an aria-label. NOTE: Only the description will be read by screen-readers (not the number). |

```

Comment:
To me, `description` still sounds a little... optional. Wondering if `text` might be more appropriate? That would line up nicely with `<d2l-button-icon>`, which is kind of a similar component.